pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -278,6 +278,11 @@ type CephVersionSpec struct {
 	// Whether to allow unsupported versions (do not set to true in production)
 	// +optional
 	AllowUnsupported bool `json:"allowUnsupported,omitempty"`
+
+	// ImagePullPolicy describes a policy for if/when to pull a container image
+	// One of Always, Never, IfNotPresent.
+	// +optional
```

Comment:
Since the schema generation didn't automatically create the enum, let's add it with this property:
```suggestion
	// +kubebuilder:validation:Enum=IfNotPresent;Always;Never;""
	// +optional
```